### PR TITLE
Monitored service at project level

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Comment by SudheendraKatte** [comment]
@shankaranandh , please review and provide comments.

File: docs/platform/monitored-service.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+When you create a monitored service at the Project Level, it becomes accessible across all the three modules: CD, SRM, and CET. 
+
+To create a monitored service at the project level:
+
+1. In Harness, select **Project** > **Monitored Services** > **+ New Monitored Service**.  
+   The Create new monitored service appears.
+
+   <docimage path={require('./static/create-monitored-service-project-level.png')} />
+
+2. In the Overview section, enter the following information, and then select **Save**:
```
Comment (brian-f-harness) [inline docs/platform/monitored-service.md:28]:
```suggestion
2. In the **Overview** section, enter the following information:
```

File: docs/platform/monitored-service.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+- at the Project level
+- within the CD module
+- inside the SRM module
+- within the CET module
+
+
+## Create a monitored service at project level
+
+When you create a monitored service at the Project Level, it becomes accessible across all the three modules: CD, SRM, and CET.
```
Comment (brian-f-harness) [inline docs/platform/monitored-service.md:19]:
```suggestion
When you create a monitored service at the Project Level, it becomes accessible across all three modules: CD, SRM, and CET. 
```

File: docs/platform/monitored-service.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+
+## Create a monitored service at project level
+
+When you create a monitored service at the Project Level, it becomes accessible across all the three modules: CD, SRM, and CET. 
+
+To create a monitored service at the project level:
+
+1. In Harness, select **Project** > **Monitored Services** > **+ New Monitored Service**.  
+   The Create new monitored service appears.
```
Comment (brian-f-harness) [inline docs/platform/monitored-service.md:24]:
```suggestion
   The **Create new monitored service** dialog appears.
```

File: docs/platform/monitored-service.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+   <docimage path={require('./static/create-monitored-service-project-level.png')} />
+
+2. In the Overview section, enter the following information, and then select **Save**:
+   
+    - **Type**: Select the type of service that you want to monitor. You can choose either Application or Environment.
+    
+    - **Create or select a Service**: You can either select an available service from the drop-down list or create a new one. To create a new service, select + Add New, enter a name for the service, an optional description, and a tag, and then select **Save**.
```
Comment (brian-f-harness) [inline docs/platform/monitored-service.md:32]:
```suggestion
    - **Create or select a Service**: You can either select an available service from the drop-down list or create a new one. To create a new service, select **+ Add New**, enter a name for the service, an optional description, and a tag, and then select **Save**.
```

File: docs/platform/monitored-service.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  
+    - **Create or select an Environment**: You can either select an available environment from the drop-down list or create a new one. To create a new environment:
+  
+        1. Select **+ Add New**.
+        
+        2. In the New Environment dialog, do the following:
```
Comment (brian-f-harness) [inline docs/platform/monitored-service.md:38]:
```suggestion
        2. In the **New Environment** dialog, do the following:
```

File: docs/platform/monitored-service.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+        
+            - Enter a name for the service, an optional description, and a tag.
+  
+            - Select an environment type. You can choose **Production** and **Non-Production**.
```
Comment (brian-f-harness) [inline docs/platform/monitored-service.md:42]:
```suggestion
            - Select an environment type. You can choose **Production** or **Non-Production**.
```

File: docs/platform/monitored-service.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+
+### View monitored service details
+
+You can view detailed information about a monitored service by clicking directly on the monitored service's name in the list. 
+
+
+#### View monitored service details from a specific module
+
+If you want to view the monitored service details within a specific module:
```
Comment (brian-f-harness) [inline docs/platform/monitored-service.md:72]:
```suggestion
To view the monitored service details within a specific module:
```

File: docs/platform/monitored-service.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+
+## Monitored service within the Service Reliability Management (SRM) module
+
+You can view and create monitored services from the SRM module.
+
+To view the monitored service list from the SRM module:
+
+1. In Harness, select **Service Reliability** > **Monitored Services**.
+   
+   A list of monitored services is displayed. The list provides information such as the service's enablement status, name, SLO and error budget details, recent changes and health trend in the past 24 hours, health score, and health status of associated dependencies.
```
Comment (brian-f-harness) [inline docs/platform/monitored-service.md:104]:
```suggestion
   A list of monitored services is displayed. The list provides information such as the service's enablement status, name, SLO and error budget details, recent changes and health trends in the past 24 hours, health score, and health status of associated dependencies.
```

**Review by brian-f-harness — APPROVED** [review]
LGTM. Left a few minor suggestions.

File: docs/platform/monitored-service.md (hunk 5 above)
Comment (SudheendraKatte) [reply docs/platform/monitored-service.md:38]: Retaining the existing format.

**Pull request merged**